Dockerfile

```diff
@@ -91,23 +89,24 @@ ENTRYPOINT ["tctl"]
 # All temporal tool binaries
 FROM alpine AS temporal-admin-tools
 
-RUN apk add --update --no-cache jq
+RUN apk add --update --no-cache \
+    jq \
+    curl \
+    ca-certificates \
+    py-pip \
+    mysql-client \
+    postgresql-client \
+    && pip install cqlsh
```

Comment:
I try to run `cqlsh` and it fails with Python error:
```
bash-5.0# cqlsh
  File "/usr/bin/cqlsh", line 121
    except ImportError, e:
                      ^
SyntaxError: invalid syntax
```
I believe it is broken for a long time or had never worked. Any ideas how to fix it?

Reply:
Seems like wrong Python version.

Reply:
vaguely remember python 2 allows the above `,`

Reply:
Ok, I fixed it.